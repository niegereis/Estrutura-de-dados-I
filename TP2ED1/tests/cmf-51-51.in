51 51
f
***************************************************
*     *           *      M*         *       *   * *
* * *** ***** *** * * * *** ******* * *** * * * * *
* * *   *   *   *   * * *   *       *   * * * *   *
* *** ***** *** ******* * * ********* *** * * *** *
*   *         *       * * * *         *   *   * * *
* * ********* ******* * *** * ********* ******* * *
* * *       *       * *   * *   *     *   *     * *
* * * * * ***** *** * *** * *** * *** *** * *** * *
* *   * * *     *   *   * *     * * *     *   *   *
******* *** ***** ***** * * ***** * ********* *****
*     * *   *   *     * * * *     *           *   *
* *** * * *** * * *** * * * * ********* * ***** ***
*   *   * *   * *   * *   * * *       *       *   *
* * *** * * *** * *** ******* ***** * * ***** *** *
* * *   *   * * * *                 * *     *   * *
*** * *** *** * *** *********** ***** ***** *** * *
*   * *       *         *       *   *         * * *
* *** *** *** ******* * * ******* *********** * * *
*   *     *     *     * *   *         * *         *
* * *** **** **** * *** *** * ***** * * * ******* *
*   *   *     *   * * * *   * *     * * *         *
*** * *** *** * *** * * * *** * ***** * *** * *****
*   *   * * *   * * * *   * * *     *   *   * *   *
* *** * * * *** * * * ***** * ***** *** * *** * * *
* * *   * *   *   * * *       *   * *   *     * * *
* * * *** * *** * * * * ******* * * ***** * *** * *
* *   *   *     * * * *   *     * * *   * * *   * *
* ***** ********* * * *** ******* * * * * * *** ***
*     *   *     * * *             *   *   *   *   *
* *** *** * *** * * *********** ********* *** * * *
*   *   * * * * * *       *     *       *     * * *
*** * *** * * * * ******* ******* ***** *** ***** *
*   * *   * *   *       *   *     *   *   *       *
* *** * *** * ******* * *** * ***** ***** ******* *
* * *   *   *   *     * *   * *     *         *   *
* * ******* *** * ***** * * * *** * * ******* * ***
* *         *   * *   * * * *   * * * *     * * * *
* *** ******* *** * * *** ***** *** * * ***** * * *
*     *     * *   * *     *   *     * * *     *   *
******* * *** * * * ******* * *** *** * * ******* *
*       *   *   * *   *     *     *   * *       * *
* ********* ***** *** * *********** *** *** ***** *
*     *   *   *   * *   *         * *   *   *     *
* ***** * * * * *** ************* * *** * *** * * *
* *     *   * *   *               *   *   *   * * *
* * ************* * ***** ******* *** ***** *** * *
* *     *   *   * * *     *       * *     * *     *
* ***** * * * * * *** ***** ******* ***** * * *** *
*     *   *   *       *                 *   *      
***************************************************
