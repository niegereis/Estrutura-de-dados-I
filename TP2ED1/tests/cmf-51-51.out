1071
***************************************************
*ooooo*ooooooooooo*ooooooM*ooooooo  *ooooooo*ooo*o*
*o*o***o*****o***o*o*o*o***o******* *o***o*o*o*o*o*
*o*o*ooo*ooo*ooo*ooo*o*o*ooo*       *ooo*o*o*o*ooo*
*o***o*****o***o*******o*o*o*********o***o*o*o***o*
*ooo*ooooooooo*ooooooo*o*o*o*ooooooooo*ooo*ooo*o*o*
*o*o*********o*******o*o***o*o*********o*******o*o*
*o*o*ooooooo*ooooooo*o*ooo*o*ooo*ooooo*ooo*ooooo*o*
*o*o*o*o*o*****o***o*o***o*o***o*o***o***o*o***o*o*
*o*ooo*o*o*ooooo*ooo*ooo*o*ooooo*o*o*ooooo*ooo*ooo*
*******o***o*****o*****o*o*o*****o*o*********o*****
*ooooo*o*ooo*ooo*ooooo*o*o*o*ooooo*ooooooooooo*ooo*
*o***o*o*o***o*o*o***o*o*o*o*o*********o*o*****o***
*ooo*ooo*o*ooo*o*ooo*o*ooo*o*o*ooooooo*ooooooo*ooo*
*o*o***o*o*o***o*o***o*******o*****o*o*o*****o***o*
*o*o*ooo*ooo*o*o*o*ooooooooooooooooo*o*ooooo*ooo*o*
***o*o***o***o*o***o***********o*****o*****o***o*o*
*ooo*o*ooooooo*ooooooooo*ooooooo*o  *ooooooooo*o*o*
*o***o***o***o*******o*o*o*******o***********o*o*o*
*ooo*ooooo*ooooo*ooooo*o*ooo*  ooooooo* *ooooooooo*
*o*o***o****o****o*o***o***o* *****o*o* *o*******o*
*ooo*ooo*ooooo*ooo*o* *o*ooo* *ooooo*o* *ooooooooo*
***o*o***o***o*o***o* *o*o*** *o***** * ***o*o*****
*ooo*ooo*o*o*ooo*o*o* *ooo* * *ooooo*   *ooo*o*ooo*
*o***o*o*o*o***o*o*o* ***** * *****o*** *o***o*o*o*
*o*o*ooo*o*ooo*ooo*o* *       *   *o*   *ooooo*o*o*
*o*o*o***o*o***o*o*o* * ******* * *o*****o*o***o*o*
*o*ooo*ooo*ooooo*o*o* *   *     * *o*ooo*o*o*ooo*o*
*o*****o*********o*o* *** ******* *o*o*o*o*o***o***
*ooooo*ooo*ooooo*o*o*             *ooo*ooo*ooo*ooo*
*o***o***o*o***o*o*o*********** *********o***o*o*o*
*ooo*ooo*o*o*o*o*o*ooooooo*     *       *ooooo*o*o*
***o*o***o*o*o*o*o*******o******* ***** ***o*****o*
*ooo*o*ooo*o*ooo*ooooooo*ooo*     *   *   *ooooooo*
*o***o*o***o*o*******o*o***o* ***** ***** *******o*
*o*o*ooo*ooo*ooo*ooooo*o*ooo* *    o*         *ooo*
*o*o*******o***o*o*****o*o*o* *** *o* ******* *o***
*o*ooooooooo*ooo*o*ooo*o*o*o*ooo* *o* *     * *o*o*
*o***o*******o***o*o*o***o*****o***o* * ***** *o*o*
*ooooo*     *o*ooo*o*ooooo*ooo*ooooo* * *     *ooo*
******* * ***o*o*o*o*******o*o***o*** * * *******o*
*       *   *ooo*o*ooo*ooooo*ooooo*   * *       *o*
* ********* *****o***o*o*********** *** *** *****o*
*     *   *   *ooo*o*ooo*ooooooooo* *   *   *ooooo*
* ***** * * * *o***o*************o* *** * ***o*o*o*
* *ooooo*   * *ooo*ooooooooooooooo*   *   *  o*o*o*
* *o*************o*o*****o*******o*** ***** ***o*o*
* *ooooo*ooo*ooo*o*o*ooooo*ooooooo* *     * *  ooo*
* *****o*o*o*o*o*o***o*****o******* ***** * * ***o*
*     *ooo*ooo*ooooooo*    o            *   *    oo
***************************************************
